test/runnable/cppa.d

```diff
@@ -983,6 +983,61 @@ void testeh3()
 }
 
 /****************************************/
+// 15579
+
+extern (C++)
+{
+    class Base
+    {
+        //~this() {}
+        void based() { }
+        ubyte x = 4;
+    }
+
+    interface Interface
+    {
+        int MethodCPP();
+        int MethodD();
+    }
+
+    class Derived : Base, Interface
```

Comment:
I guess, D compiler doesn't add pointer to the typeid(Derived) into its vtbl.
Maybe we should add pointer to the C++-generated RTTI as external symbol? Now we know how to mangle C++ type_info and we know that Derived defined in C++ code.